src/main/java/formflow/library/data/UserFile.java

```diff
@@ -30,6 +31,7 @@
 
 @Entity
 @Table(name = "user_files")
+@DynamicInsert
```

Comment:
This is interesting - do we need this on the submission table as well?

Reply:
We need it if we are relying on the database to supply defaults.